### ISSUE TITLE
URL parser breaking when http is missing

### ISSUE THREAD

**Issue by kidwellj** [mapcomm/common-ground_map#23]
The parser renders a malformed URL for fields which are missing http://. For example, "st. Andrews cathedral" in Inverness renders as http://localhost:8081/www.invernesscathedral.co.uk. We also need to check for handling of https:// when it is present in field.

**Comment by danhillcode** [comment]
can I write a script to format the data already in the database or provide a csv of these points and I can add http to them so they are formatted the same as the other sets.

**Comment by kidwellj** [comment]
Yeah, we can definitely fix the database with a quick parse. The reason I haven't defaulted to this is that that Carto doesn't have any data checking capabilities, so it's just entry for a text string for the URL field, so future users may introduce errors, especially if we aren't monitoring all the tables closely. I agree that this issue is trivial overall though, so let's not waste any dev time on this fix. I'll change to an "enhancement" for later.
